site/content/en/_index.html

```diff
@@ -0,0 +1,89 @@
+---
+title: "minikube"
+resources:
+- src: "logo.png"
+  title: "logo"
+
+- src: "start.jpg"
+  title: "start"
+---
+{{< blocks/cover title="Welcome to minikube" image_anchor="top" height="full" color="orange" >}}
+<div class="mx-auto">
+	<a class="btn btn-lg btn-primary mr-3 mb-4" href="{{< relref "/docs" >}}">
+		Read the docs! <i class="fas fa-arrow-alt-circle-right ml-2"></i>
+	</a>
+	<a class="btn btn-lg btn-secondary mr-3 mb-4" href="https://github.com/google/docsy-example">
+		Download <i class="fab fa-github ml-2 "></i>
+	</a>
+	<p class="lead mt-5">The local Kubernetes environment for macOS, Linux, and Windows.</p>
+	<div class="mx-auto mt-5">
+		{{< blocks/link-down color="info" >}}
+	</div>
+</div>
+{{< /blocks/cover >}}
+
+
+{{% blocks/lead color="primary" %}}
+
+minikube is a local Kubernetes environment focused on application developers.
+
+{{< imgproc "start" Fit "730x239" >}}{{< /imgproc >}}
+
+Kubernetes certified, mother approved.
```

Comment:
Kid Tested, Mother Approved ? :-) Does this mean we need to pass the certification test again ?

Reply:
Yes =)